protocol/cosi.go

```diff
@@ -21,17 +24,16 @@ func init() {
 //  - Commitment
 //  - Challenge
 //  - Response
-// It uses lib/cosi as the main structure for the protocol.
 
-// ProtocolCosi is the main structure holding the round and the sda.Node.
-type ProtocolCosi struct {
+// CoSi is the main structure holding the round and the sda.Node.
+type CoSi struct {
 	// The node that represents us
 	*sda.TreeNodeInstance
 	// TreeNodeId cached
 	treeNodeID sda.TreeNodeID
 	// the cosi struct we use (since it is a cosi protocol)
 	// Public because we will need it from other protocols.
-	Cosi *cosi.Cosi
+	Cosi *cosi.CoSi
```

Comment:
true, here we don't need to be able to interfere with the structure. The only thing we need at the end is the signature. At least for the moment.